src/alto-ui/Actions/__tests__/Actions.test.js

```diff
@@ -0,0 +1,70 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+
+import Actions from '../Actions';
+
```

Comment:
Remove this empty line
```suggestion
```

Reply:
fixed in "fixes after code review" commit